src/main/java/FrameworksDrivers/ScheduleGraphics.java

```diff
@@ -201,20 +185,22 @@ private static void setLookAndFeel() {
         }
     }
 
-    public static void main (String[] arguments) {
 
+    public static void main (String[] arguments) {
         setLookAndFeel();
         ScheduleGraphics gf = new ScheduleGraphics();
-        gf.repaint();
+        gf.repaint(); // repaint is called so that the graphics are updated
 
     }
 
     @Override
     public void actionPerformed(ActionEvent e) {
         Object source = e.getSource();
 
+        // the following code will run if a given button is pressed
+
 
-        if (source == addEventButton) { // if addEventButton is pressed
+        if (source == addEventButton) {
             Object[] addEventText = {"Name:", addEventName, "Date:", addEventDate, "Start time:", addEventStart, "End time:", addEventEnd};
```

Comment:
I'm fairly certain we're not supposed to directly initialize objects? is there a more specific object we can replace this with?